example/profiler/README.md

```diff
@@ -5,7 +5,9 @@ Please refer to [this link](http://mxnet.incubator.apache.org/faq/perf.html?high
 for visualizing profiling results and make sure that you have installed a version of MXNet compiled
 with `USE_PROFILER=1`.
 
-- profiler_executor.py. To run this example, simply type `python profiler_executor.py` in terminal.
+- profiler_executor.py. To run this example,
+1. clone mxnet-memonger (git clone https://github.com/dmlc/mxnet-memonger.git). Add path to mxnet-memonger to PYTHONPATH
+2. type `python profiler_executor.py` in terminal.
```

Comment:
Format these lines like this - 

- To run ``profiler_executor.py`` example,
  - clone mxnet-memonger (git clone https://github.com/dmlc/mxnet-memonger.git). Add path to mxnet-memonger to PYTHONPATH
  - type `python profiler_executor.py` in terminal.
  It will generate a json file named `profile_executor_5iter.json`.

Reply:
Will make this change and resubmit. Thanks